docs/installation-docker.rst

```diff
@@ -0,0 +1,81 @@
+=======================
+Installation via Docker
+=======================
+
+Kuma has experimental support for `Docker`_ as an alternative to the
+Vagrant set up.
+
+.. _Docker: https://www.docker.com/
+
+Docker setup
+============
+
+#. Ensure you have cloned the kuma Git repository and it is your current
+   working directory::
+
+        git clone git@github.com:mozilla/kuma.git
+        cd kuma
+
+#. Install the `Docker Toolbox`_. This will install the Docker engine,
+   along with Docker Compose and Docker Machine.
+
+   .. _Docker Toolbox: https://www.docker.com/products/docker-toolbox
+
+#. Create the virtual machine that Docker will run within::
+
+        docker-machine create --driver virtualbox --virtualbox-memory 2048 kuma
```

Comment:
I could bump the default up. I think Vagrant we use 4096 so I can use that. I don't think you can resize it after the fact.